lib/jekyll/converters/coffeescript.rb

```diff
@@ -10,10 +10,10 @@ def setup
       end
 
       def matches(ext)
-        ext.downcase == ".coffee"
+        ext.casecmp(".coffee").zero?
```

Comment:
Yuck, but ok…

Reply:
https://rubocop.readthedocs.io/en/latest/cops_performance/#performancecasecmp

Reply:
TIL

Reply:
It doesn't read as clearly, but it's faster.